homeassistant/components/isy994/light.py

```diff
@@ -65,6 +66,9 @@ def brightness(self) -> float:
         """Get the brightness of the ISY994 light."""
         if self._node.status == ISY_VALUE_UNKNOWN:
             return None
+        # Special Case for ISY Z-Wave Devices using % instead of 0-255:
+        if self._node.uom == UOM_PERCENTAGE:
+            return int(self._node.status * 255.0 / 100.0)
```

Comment:
Should we use `round()` here?

Reply:
Probably not a big difference (e.g. 127 vs 128 for 50%) but I changed it. I also updated `cover.py` to be consistent.